WordPress/src/test/java/org/wordpress/android/util/SnackbarSequencerConcurrentTest.kt

```diff
@@ -76,42 +81,91 @@ class SnackbarSequencerConcurrentTest {
         // Then
         verify(wpSnackbar, times(1)).show()
         advanceTimeBy(checkPoints[0])
-        verify(wpSnackbar, times(1)).show()
+        verify(wpSnackbar, times(2)).show()
         advanceTimeBy(checkPoints[1])
+        verify(wpSnackbar, times(3)).show()
+        advanceTimeBy(checkPoints[2])
+
+        // Nothing else shown
+        verify(wpSnackbar, times(3)).show()
+        verifyNoMoreInteractions(wpSnackbar)
+    }
+
+    @Test
+    fun `snackbars are not shown until previous duration elapsed`() = runBlockingTest(coroutineContext) {
+        // Given
+        val items = getItems()
+        val checkPoints = getCheckPoints(items)
+
+        // When
+        for (item in items) {
+            sequencer.enqueue(item)
+        }
+
+        // Then
+        verify(wpSnackbar, times(1)).show()
+        advanceTimeBy(checkPoints[0] - 2 * SNACKBAR_DURATION_MARGIN)
+        verify(wpSnackbar, times(1)).show()
+        advanceTimeBy(2 * SNACKBAR_DURATION_MARGIN)
         verify(wpSnackbar, times(2)).show()
```

Comment:
We might want to consider removing all the lines below this line (exclusive). I think we are technically testing the same thing as "snackbars are shown in sequence with the correct duration".It would make the purpose of the test clearer imo. Wdyt?

Reply:
Well, I think you are right 😊! Let me know what you think of my answers to the previous points and I will change also this one as you suggested, thanks 🙇‍♂️!

Reply:
Maybe we can consider this addressed [here](https://github.com/wordpress-mobile/WordPress-Android/pull/10856#discussion_r351933046)?